hera_sim/interpolators.py

```diff
@@ -0,0 +1,99 @@
+"""
+This module provides interfaces to different interpolation classes.
+"""
+
+import numpy as np
+from scipy.interpolate import RectBivariateSpline
+
+class Tsky:
+    """
+    # TODO: fill in docstring
+    """
+    def __init__(self, datafile, **interp_kwargs):
+        self.datafile = datafile
+        self._data = np.load(self.datafile, allow_pickle=True)
+        # make sure the correct keys are used
+        assert 'freqs' in self._data.keys(), \
+                "The frequencies corresponding to the sky temperature array " \
+                "must be provided. They must be saved to the npz file using " \
+                "the key 'freqs'."
+        assert 'lsts' in self._data.keys(), \
+                "The LSTs corresponding to the sky temperature array must " \
+                "be provided. They must be saved to the npz file using the " \
+                "key 'lsts'."
+        assert 'tsky' in self._data.keys(), \
+                "The sky temperature array must be saved to the npz file " \
+                "using the key 'tsky'."
+        assert 'meta' in self._data.keys(), \
+                "The npz file must contain a metadata dictionary that can " \
+                "be accessed with the key 'meta'. This dictionary should " \
+                "provide information about the units of the various arrays " \
+                "and the polarizations of the sky temperature array."
+        # now make sure that the sky temperature array has the correct shape
```

Comment:
@steven-murray do you think it makes sense and is a good idea to require the `npz` files to conform to this structure? That is, does it make sense to require that `npz` files also contain a `meta` dictionary that contains information about the arrays' units and the polarizations included and to require the sky temperature arrays to have `shape==(NPOLS,NLSTS,NFREQS)`?

Also, do you see any clear benefits of implementing `Tsky` more in-line with what you proposed rather than the way I've implemented it here?

Reply:
This looks pretty good. I like the idea of a `meta` dictionary in general, though to be honest, I'd just as well have the `.npz` arrays have an exact specific form here, readable by this specific class, and always interpreted the same way. I think it's overkill to allow for unit conversion etc here at this point. If these `.npz` files were going to be created a lot and by lots of different people, maybe you'd want that. But really, we're going to have a pretty standard/static library of these. That means that `meta` really just comes down to specifying pols. 

Forcing the sky temperature array to have that shape is good. Any well-defined standard is better than none.